default/web_tt2/ls_templates.tt2

```diff
@@ -3,196 +3,194 @@
+                                                <input type="hidden" name="webormail"     value="[% webormail %]" />
+                                                <input class="MainMenuLinks" type="submit" name="action_edit_template"   value="[%|loc%]edit[% END %]" />
+                                                <input class="MainMenuLinks" type="submit" name="action_copy_template"   value="[%|loc%]cp[% END %]" />
+                                                <input class="MainMenuLinks" type="submit" name="action_remove_template" value="[%|loc%]rm[% END %]" />
+                                            </fieldset>
+                                        </form>
+                                    [% END %]
+                                </td>
+                            [% END %]
+                        [% END %]
+
+                        [% IF list %]
+                            [% FOREACH lang = file.value.list %]
+                                <td class="text_center">
+                                    [% IF lang.value %]
+                                        <form action="[% 'edit_template' | url_rel %]" method="post">
+                                            <fieldset>
+                                                <input type="hidden" name="template_name" value="[% file.key %]" />
+                                                <input type="hidden" name="template_path" value="[% lang.value %]" />
+                                                <input type="hidden" name="scope"         value="list" />
+                                                <input type="hidden" name="tpl_lang"      value="[% lang.key %]" />
+                                                <input type="hidden" name="list"          value="[% list %]" />
+                                                <input type="hidden" name="webormail"     value="[% webormail %]" />
+                                                <input class="MainMenuLinks" type="submit" name="action_edit_template"   value="[%|loc%]edit[% END %]" />
+                                                <input class="MainMenuLinks" type="submit" name="action_copy_template"   value="[%|loc%]cp[% END %]" />
+                                                <input class="MainMenuLinks" type="submit" name="action_remove_template" value="[%|loc%]rm[% END %]" />
+                                            </fieldset>
+                                        </form>
+                                    [% END %]
+                                </td>
+                            [% END %]
+                        [% ELSE %]
+                            <td></td>
+                        [% END %]
+                    </tr>
+            [% END %]
+    [% IF templates %]
+        </table>
     [% END %]
-    </td>
-    [%END%]
-  [% ELSE %]
-  <td></td>
-  [% END %]
-[% END %]
-[% IF templates %]
-</td></tr>
-</table>
-[% END %]
 </p>
 <!-- end ls_templates.tt2 -->
```

Comment:
I'll check this file afterward again.

Reply:
This one is quite strange. I mean, why having `<table>` only if `templates` but always having table rows. I think we should always put `<table>` and have the caption and first row only if `templates`.

What do you think?

Reply:
I feel strange, too.  Perhaps `templates` is always assigned.  I'll check the code of wwsympa when I'll wake up.

Thanks for your bold work!

Reply:
I see `templates` is always assigned (except in case of very mulformed input).   [`[% END %]`](https://github.com/sympa-community/sympa/blob/3aea9d01cb857cc7d42cae0119ee8275d9a0fea2/default/web_tt2/ls_templates.tt2#L55) at line 55 and [`[% IF templates %]`](https://github.com/sympa-community/sympa/blob/3aea9d01cb857cc7d42cae0119ee8275d9a0fea2/default/web_tt2/ls_templates.tt2#L192) at line 192 should be removed.

Reply:
Do you prefer to do it or me? Since you’re basing #592 on this PR, if I do it, you’ll have to pull the commit into your branch.

Reply:
Please do it on this PR. I’ll rebase your fix on my PR.

Reply:
Fixed in https://github.com/sympa-community/sympa/pull/582/commits/fd9c959deffe332ae9ddda65efa7bd0a9f21af14

Reply:
Confirmed.